src/model/backup.ts

```diff
@@ -232,7 +246,40 @@ async function restoreCurrentBackupIntoKeychain(
         if (endsWith(key, seedPhraseKey) || endsWith(key, privateKeyKey)) {
           accessControl = privateAccessControlOptions;
         }
-        if (typeof value === 'string') {
+
+        const hasBiometricsEnabled = await getSupportedBiometryType();
+
+        if (!hasBiometricsEnabled && android && endsWith(key, seedPhraseKey)) {
+          const parsedValue = JSON.parse(value);
+          const { seedphrase } = parsedValue;
+
+          const isBackupWasSavedWithPIN =
```

Comment:
is... was...? Grammar in this variable name went crazy 😅.

Maybe `wasBackupSavedWithPIN`?